libs/payments/ui/src/lib/actions/index.ts

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Use this file to export React client components (e.g. those with 'use client' directive) or other non-server utilities
```

Comment:
This comment doesn't seem relevant here. Copy mistake?

Reply:
I wasn't sure why it was in the original location either, happy to remove!

Reply:
Client and Server components/functions/etc. shouldn't be exported in the same barrel file. The comment is just to indicate that a file should be only for client or server resources.